src/pages/edge/adobe-journey-optimizer-decisioning/release-notes.md

```diff
@@ -8,39 +8,49 @@ keywords:
 
 # Release notes
 
+## February 6, 2025
+
+### Adobe Journey Optimizer - Decisioning extension 1.1.0
+
+Added configuration capability to the Adobe Journey Optimizer - Decisioning extension in the Launch UI for mobile Tag Properties.
+
+Users can now configure:
+
+* Request timeout - Set the amount of time (in seconds) to wait for a response from the decisioning services before timing out.
+
 ## December 6, 2024
 
 ### Android Optimize 3.2.2
 
-* Added timeout (in seconds) parameter to updatePropositions and getPropositions APIs to enable timeout configuration in both the apis ensuring that the operation either completes within the given time frame or returns an error indicating a timeout.
-* Updated getPropositions API to return cached propositions immediately if there are no pending updatePropositions calls for the requested list of decision scopes.
-* Added support for debug events and enabled getPropositions and onPropositionsUpdate to receive Simulated events from Assurance.
+* Added timeout (in seconds) parameter to updatePropositions and getPropositions APIs to enable timeout configuration in both the apis ensuring that the operation either completes within the given time frame or returns an error indicating a timeout.
+* Updated getPropositions API to return cached propositions immediately if there are no pending updatePropositions calls for the requested list of decision scopes.
+* Added support for debug events and enabled getPropositions and onPropositionsUpdate to receive Simulated events from Assurance.
```

Comment:
Is "Simulated" a proper noun here? I'd just go with "... to receive simulated events..." if not.